distribution/tools/server-cli/src/main/java/org/elasticsearch/server/cli/SystemJvmOptions.java

```diff
@@ -138,6 +138,9 @@ private static Stream<String> maybeWorkaroundG1Bug() {
     }
 
     private static Stream<String> maybeAllowSecurityManager() {
+        if (Runtime.version().feature() >= 24) {
```

Comment:
Would it make sense to replace this `Runtime.version().feature()` check all over the place with some single static?

Reply:
That's a good point, I'll do that if I find a common place to put it.
But keep in mind all of this is temporary and won't make it into 9.0 or 8.18 - we'll make another pass and remove those completely, along with any trace of SM.

Reply:
Done!